### PR TITLE
updates with latest nightly v0.0.0-20240201git319eebd git ref 319eebdb81

### PR DESCRIPTION
**Pull request by Deflaimun**

### CODE REVIEW

**Comment by Deflaimun** [comment]
Doc PR
https://github.com/redpanda-data/docs/pull/286

File: tools/gen/23.3/rpk-registry-schema-get.adoc [hunk 1]
```diff
@@ -8,7 +8,9 @@ potential (mutually exclusive) ways:
 
 * By version, returning a schema for a required subject and version
```
Comment (JakeSCahill) [inline tools/gen/23.3/rpk-registry-schema-get.adoc:9]: What are these ^M characters?

**Review by JakeSCahill — APPROVED** [review]

File: tools/gen/23.3/rpk-registry-schema-get.adoc (hunk 1 above)
Comment (Deflaimun) [reply tools/gen/23.3/rpk-registry-schema-get.adoc:9]: Not sure. But they were already present in the file. I'm guessing there's something to do with Windows formatting as opposed to posix formatting.

Comment (Deflaimun) [reply tools/gen/23.3/rpk-registry-schema-get.adoc:9]: Yup. Difference in EOL/New Line formatts
https://stackoverflow.com/questions/1889559/make-git-diff-ignore-m

**Pull request merged**